astro/src/content/quickstarts/quickstart-php-laravel-web.mdx

```diff
@@ -0,0 +1,303 @@
+```
+
+Edit `your-application/.env` and insert the following lines.
+
+```
+FUSIONAUTH_CLIENT_ID="E9FDB985-9173-4E01-9D73-AC2D60D1DC8E"
+FUSIONAUTH_CLIENT_SECRET="super-secret-secret-that-should-be-regenerated-for-production"
+FUSIONAUTH_BASE_URL="http://fusionauth:9011"
+FUSIONAUTH_REDIRECT_URL="http://localhost:3000/auth/callback"
+# FUSIONAUTH_TENANT_ID=
+```
+
+This tells {frontmatter.technology} where to find and connect to FusionAuth.
+
+In this file, also overwrite the following database fields.
+
+```
+DB_HOST=mariadb
+DB_USERNAME=bn_myapp
+DB_DATABASE=bitnami_myapp
+```
+
+This enables {frontmatter.technology} to connect to the database running in the adjacent Docker container.
+
+<Aside type="note">
+  Note that {frontmatter.technology} will silently overwrite the `DB_HOST` value with `mariadb` in the file `InteractsWithDockerComposeServices.php`, so you can't change this value or the database service name in Docker. If you encounter database connection errors in your own apps running in Docker, check that this file isn't interfering with your connection strings.
+</Aside>
+
+In the file `your-application/config/services.php`, add one more item to the array being returned with the code below.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/config/services.php"}
+  lang="php"
+  tags="fusionauth" />
+
+Now the configuration files have all the settings you need. Next, you'll change your provider to be aware of FusionAuth.
+
+In the file `your-application/app/Providers/EventServiceProvider.php`, replace the `$listen` array with the code below.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/app/Providers/EventServiceProvider.php"}
+  lang="php"
+  tags="fusionauth" />
+
+Finally, update your User model to have FusionAuth fields.
+
+In the file `your-application/app/Models/User.php`, add the FusionAuth items to the `$fillable` and `$hidden` properties with the code below.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/app/Models/User.php"}
+  lang="php"
+  tags="fusionauth" />
+
+Every model in {frontmatter.technology} uses the database, so you need to update the database table to match your new model.
+
+Create a file named `your-application/database/migrations/2023_09_27_142009_add_fusionauth_fields_user_table.php` or run the command below to create it.
+
+```shell
+docker compose exec lara_app php artisan make:migration add_fusionauth_fields_user_table
+```
+
+The result is the same whether you create the database scripts manually or with this command, so you can use your preferred method.
+
+Add the code below to the file.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/database/migrations/2023_09_27_142009_add_fusionauth_fields_user_table.php"}
+  lang="php" />
+
+Run the command below to update your MariaDB database with the new User definition.
+
+```shell
+docker compose exec lara_app php artisan migrate
```

Comment:
I am getting an error here and can't continue.  Any ideas?

Reply:
@mark-robustelli could you share the log of the error you are getting. The migration succeeds for me and I was not able to reproduce the error.

![CleanShot 2023-10-18 at 15 47 31@2x](https://github.com/FusionAuth/fusionauth-site/assets/31291528/7c434984-a13b-4367-a65a-222d5262a7e1)

Reply:
@rideam ,  I apologize for not posting the results before.  Here it is:

 ```
Illuminate\Database\QueryException 

  SQLSTATE[HY000] [1045] Access denied for user 'bn_myapp'@'192.168.80.4' (using password: YES) (Connection: mysql, SQL: select * from information_schema.tables where table_schema = laravel and table_name = migrations and table_type = 'BASE TABLE')

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:801
    797▕                     $this->getName(), $query, $this->prepareBindings($bindings), $e
    798▕                 );
    799▕             }
    800▕ 
  ➜ 801▕             throw new QueryException(
    802▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    803▕             );
    804▕         }
    805▕     }

      +35 vendor frames 

  36  artisan:35
      Illuminate\Foundation\Console\Kernel::handle()
```

Reply:
@mark-robustelli from the log you provided it seems like the database connection environment variables values in `your-application/.env` file are not set correctly. 

Can you check if you set the values for these keys correctly in the file `your-application/.env`. Note you need to overwrite the default values that are already set up in that file as described in the **Authentication** section.

```
DB_HOST=mariadb
DB_USERNAME=bn_myapp
DB_DATABASE=bitnami_myapp
```

the log should be pointing to the `bitnami_myapp` schema and not the `laravel` schema if those values are set correctly

![CleanShot 2023-10-18 at 23 39 15@2x](https://github.com/FusionAuth/fusionauth-site/assets/31291528/e2539f8c-af27-4d29-b86a-a2b1f2f22806)